src/os/filestore/FileStore.cc

```diff
@@ -3756,7 +3756,7 @@ int FileStore::_clone_range(const coll_t& cid, const ghobject_t& oldoid, const g
 /*
  * Move contents of temp object according to move_info to base object. Once the move_info is traversed completely, delete the temp object.
  */
-int FileStore::_move_tempobjs(const coll_t& temp_cid, const ghobject_t& temp_oid, const coll_t& cid, const ghobject_t& oid,
+int FileStore::_move_tempobjs(const coll_t& src_cid, const ghobject_t& src_oid, const coll_t& cid, const ghobject_t& oid,
```

Comment:
one is tempobj [src_oid] and other is base object [oid]. we are moving src_oid --> oid and then deleting the src_oid.